lib/elixir/lib/calendar/iso.ex

```diff
@@ -289,33 +342,64 @@ defmodule Calendar.ISO do
   """
   @doc since: "1.10.0"
   @impl true
-  def parse_date("-" <> string) when is_binary(string),
-    do: parse_date(string, -1)
+  def parse_date(string) when is_binary(string),
+    do: parse_date(string, :extended)
```

Comment:
Shall we use default arguments here instead? :)

Reply:
I didn't use defaults so that 1) we can have different docs/examples for each, and 2) we could have separate `@since` tags, but if neither of those are valuable I can rewrite them into defaults!

Reply:
Ah, the since tag is a good one. But I think the docs are actually worse spread apart. So maybe just a short doc saying it is equivalent to /2 with extended? (If that’s already not the case)

Reply:
That's pretty much what I have in mind: `parse_string/1` with common `:extended` usage docs and `parse_string/2` showing `:basic`. Here's what it looks like rendered locally:

![Screen Shot 2021-02-08 at 11 27 06 AM](https://user-images.githubusercontent.com/1406220/107271097-9c508900-6a00-11eb-85d9-1e994b9d949e.png)

Reply:
(The `ISO 8601` link takes you up to the extensive moduledoc section we've added previously)
![Screen Shot 2021-02-08 at 11 33 04 AM](https://user-images.githubusercontent.com/1406220/107271785-90b19200-6a01-11eb-8a67-d218148805d1.png)